independent-publisher-connectors/Microsoft Learn Catalog/apiDefinition.swagger.json

```diff
@@ -0,0 +1,780 @@
+                    }
+                  },
+                  "description": "roles"
+                },
+                "subjects": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "string",
+                        "description": "id"
+                      },
+                      "name": {
+                        "type": "string",
+                        "description": "name"
+                      },
+                      "children": {
+                        "type": "array",
+                        "items": {
+                          "type": "object",
+                          "properties": {
+                            "id": {
+                              "type": "string",
+                              "description": "id"
+                            },
+                            "name": {
+                              "type": "string",
+                              "description": "name"
+                            }
+                          }
+                        },
+                        "description": "children"
+                      }
+                    }
+                  },
+                  "description": "subjects"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Learning Content",
+        "operationId": "GetLearningContent",
+        "description": "This connector can return information from the Microsoft Learn Catalog about all publicly available Modules, Units, Learning Paths, Instructor-Led Courses, Exams, Certifications.",
+        "parameters": [
+          {
+            "name": "locale",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "A single, valid locale code from the supported list of locales. The returned metadata will be in the requested locale if available. If this parameter isn't supplied, the en-us response will be returned. Example: en-us",
+            "x-ms-summary": "Locale"
+          },
+          {
+            "name": "type",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "x-ms-summary": "Type",
+            "description": "A comma-separated list of one or more of the top-level content or taxonomies objects we currently provide in the response to return. Supported values are: modules, units, learningPaths, certifications, exams, courses, levels, roles, products, subjects. Example: modules,learningPaths"
+          },
+          {
+            "name": "level",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "A comma-separated list of one or more of the levels we currently have available. Supported values are: beginner, intermediate, advanced.. Example: beginner",
```

Comment:
@SeanKe11y all files should be free of grammatical errors, syntax errors and typos. please remove the double periods in the description. You have "Supported values are: beginner, intermediate, advanced.." please change to "Supported values are: beginner, intermediate, advanced."

Reply:
Removed